staging/src/k8s.io/api/admissionregistration/v1/generated.pb.go

```diff
@@ -342,13 +342,13 @@ func init() {
 }
 
 var fileDescriptor_aaac5994f79683e8 = []byte{
```

Comment:
these changes are concerning. Do we use different protoc?

Reply:
Unfortunately, we do upstream uses something newer, but we are still at 3.0.0. I'll add that as a followup task.